contrib/drivers/filesharedrivers/nfs/cli.go

```diff
@@ -0,0 +1,237 @@
+//    Copyright 2019 The OpenSDS Authors.
```

Comment:
Please update the copyright style.

Reply:
//    Copyright (c) 2019 The OpenSDS Authors., Ltd. All Rights Reserved.

I hope above copyright style is correct

Reply:
No, it should be `// Copyright 2019 The OpenSDS Authors.`, only one blank space required.